src/plugins/faultManagement/fault-manager-inspector.scss

```diff
@@ -0,0 +1,3 @@
+.c-fault-mgmt__inspector__values{
+    background: rgb(44, 2, 6) !important;
+}
```

Comment:
new line missing here.

Reply:
Deleting this file all together. Its not being used for v1.